KalturaMediaServer/src/main/java/com/kaltura/media/server/managers/KalturaLiveManager.java

```diff
@@ -491,6 +504,22 @@ public void onEvent(IKalturaEvent event){
 		}
 	}
 
+	protected  void onStreamReadyForPlayback(String entryId, final KalturaMediaServerIndex serverIndex) {
+		logger.debug("onStreamReadyForPlayback [" + entryId + "]");
+
+		LiveEntryCache liveEntryCache = null;
```

Comment:
u add this var here, but onlu use it inside.
u can just do entries.get(entryId).setEntryIsLive(); in line 515 instead this line and instead of 515-516